conf.yaml

```
@@ -748,7 +749,8 @@ contents:
                 repo:   clients-team
                 path:   docs/examples/elastic-cloud/php
                 map_branches: &mapCloudEceToClientsTeam
-                  release-ms-41: master
+                  ms-47: master
+                  2.6: master
```

```suggestion
                  release-ms-41: master
```
Same as above, we need to use the branch name here.